From 69206419f705e92d5d8b04b20846d68641901271 Mon Sep 17 00:00:00 2001
From: hscham <hscham@chromium.org>
Date: Mon, 30 Nov 2020 16:39:17 +0900
Subject: [PATCH] libchrome: Use CompareCase::SENSITIVE as default
 case_sensitivity

For base::{Starts,Ends}With, which would be used in place of deprecated
BasicStringPiece::{starts,ends}_with.

Change-Id: Ib38bb88c58a1858e7f9c86b4956966a2aac3b82e
---
 base/strings/string_util.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/base/strings/string_util.h b/base/strings/string_util.h
index e5a0487cff..859b714328 100644
--- a/base/strings/string_util.h
+++ b/base/strings/string_util.h
@@ -315,16 +315,16 @@ enum class CompareCase {
 
 BASE_EXPORT bool StartsWith(StringPiece str,
                             StringPiece search_for,
-                            CompareCase case_sensitivity);
+                            CompareCase case_sensitivity = CompareCase::SENSITIVE);
 BASE_EXPORT bool StartsWith(StringPiece16 str,
                             StringPiece16 search_for,
-                            CompareCase case_sensitivity);
+                            CompareCase case_sensitivity = CompareCase::SENSITIVE);
 BASE_EXPORT bool EndsWith(StringPiece str,
                           StringPiece search_for,
-                          CompareCase case_sensitivity);
+                          CompareCase case_sensitivity = CompareCase::SENSITIVE);
 BASE_EXPORT bool EndsWith(StringPiece16 str,
                           StringPiece16 search_for,
-                          CompareCase case_sensitivity);
+                          CompareCase case_sensitivity = CompareCase::SENSITIVE);
 
 // Determines the type of ASCII character, independent of locale (the C
 // library versions will change based on locale).
-- 
2.29.2.454.gaff20da3a2-goog

